core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5646,27 +5650,151 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 #endif
                 goto call_func_from_entry;
             }
-#if WASM_ENABLE_SIMD != 0
+#if WASM_ENABLE_SIMD != 0 && WAMR_BUILD_SIMDE != 0
```

Comment:
again, not sure if it ever makes sense to enable SIMDE without enabling SIMD, so we could probably define those flags in a way that only SIMDE check here is required